src/extension.ts

```diff
@@ -177,8 +184,8 @@ async function removeMongoServer(node: INode) {
 	}
 }
 
-function dropDatabase(database: MongoDatabaseNode): void {
-	vscode.window.showInformationMessage('Are you sure you want to drop the database \'' + database.id + '\' and its collections?', { modal: true }, 'Drop')
+function deleteDatabase(database: MongoDatabaseNode): void {
+	vscode.window.showInformationMessage('Are you sure you want to delete database \'' + database.id + '\' and its collections?', { modal: true }, 'Drop')
```

Comment:
Can you change this message to be the same as your other messages?
1. warning instead of info
1. not modal
1. 'yes' and 'no' instead of 'drop'